.travis.yml

```diff
@@ -16,21 +15,24 @@ matrix:
 
   include:
 
-  - env: STACK_YAML="$HOME/build/serokell/log-warper/stack.yaml"
+  - ghc: 8.2.2
+    env: STACK_YAML="$HOME/build/serokell/log-warper/stack.yaml"
     os: linux
     addons:
       apt:
         packages:
         - libgmp-dev
 
-  - env: STACK_YAML="$HOME/build/serokell/log-warper/stack-8.0.2.yaml"
+  - ghc: 8.0.2
+    env: STACK_YAML="$HOME/build/serokell/log-warper/stack-8.0.2.yaml"
     os: linux
     addons:
       apt:
         packages:
         - libgmp-dev
 
-  - env: GHCVER='8.2.2' CABALVER='head'
+  - ghc: 8.2.2
```

Comment:
This is redundant, the compiler version is determined by GHCVER here.

Reply:
This is not redundant. Without this option it's harder to me to determine for which version of compiler CI is failing. Compare:

**without this option**

![screenshot from 2018-01-05 14-58-05](https://user-images.githubusercontent.com/4276606/34608441-5e8f3456-f229-11e7-8dc2-45d18c0a86f7.png)

**with this option**

![screenshot from 2018-01-05 14-57-57](https://user-images.githubusercontent.com/4276606/34608443-6463803a-f229-11e7-922f-4a23201d86bb.png)